core/src/main/java/org/n52/iceland/ogc/gml/time/TimePosition.java

```diff
@@ -62,8 +54,7 @@ public TimePosition(DateTime time) {
      *            Indeterminate value of time position
      */
     public TimePosition(TimeIndeterminateValue indeterminateValue) {
-        super();
-        this.indeterminateValue = indeterminateValue;
+        this.indeterminateValue = Optional.of(indeterminateValue);
```

Comment:
Well, then I would allow somebody to pass ``null`` to the constructor, right? What would be the advantage of that?